### PR TITLE
SCRUM-1099 changed RGD assembly name for download

### PR DESCRIPTION
**Pull request by adamgibs**

### CODE REVIEW

**Comment by nuin** [comment]
I can approve this if you want and merge. Let me know

**Comment by adamgibs** [comment]
@nuin Thanks for the offer! But there is still some backend work that needs to happen before this can be merged in, so there's no rush.

**Comment by nuin** [comment]
@adamgibs No worries, anyone in the Blue Team can help if needed. Just tag us.

**Review by markquintontulloch — APPROVED** [review]

**Pull request merged**
